packages/vsce/src/trees/CICSCombinedTrees/CICSCombinedWebServiceTree.ts

```diff
@@ -204,4 +209,8 @@ export class CICSCombinedWebServiceTree extends TreeItem {
   public getParent() {
     return this.parentPlex;
   }
+
+  public getSession() {
+    return this.getParent().getParent().getSession();
```

Comment:
Can this just be `this.getParent().getSession()` as the parent's `getSession` will also keep going up the tree??

Reply:
If so, same for all combination trees